ros2doctor/ros2doctor/verb/call.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import socket
+import struct
+import threading
+import time
+
+import rclpy
+from rclpy.executors import MultiThreadedExecutor
+from rclpy.node import Node
+from ros2doctor.verb import VerbExtension
+
+from std_msgs.msg import String
+
+DEFAULT_GROUP = '225.0.0.1'
+DEFAULT_PORT = 49150
+DEFAULT_TOPIC = '/knockknock'
+summary_table = {}
+
+
+class CallVerb(VerbExtension):
+    """Publish and subscribe, multicast send and receive, print table."""
```

Comment:
I suggest making this description a bit more verbose so it's clear what this tool is does. E.g.

```
"""
Check network connectivity between multiple hosts.

This command can be invoked on multiple hosts to confirm that they can talk to each other.

TODO <describe how pub/sub is used to test ROS comms>
TODO <describe how multicast send/receive is used to confirm multicast UDP comms>
```